app/components/TotalDeposit.tsx

```diff
@@ -111,14 +110,14 @@ export const columns: Column<DApp>[] = [
 
 function DepositCard({ title, value }: DepositData) {
 	return (
-		<Card className="flex flex-1 max-w-1/4">
-			<CardContent className="p-6 rounded-lg text-white flex flex-col gap-2 bg-azure-10 w-full">
+		<div className="card flex flex-1 max-w-1/4 card-border border rounded-lg">
+			<div className="card-body p-6 rounded-lg text-white flex flex-col gap-2 bg-azure-10 w-full">
```

Comment:
```suggestion
			<div className="card-body p-6 rounded-lg text-white flex flex-col gap-2 w-full">
```

let's remove this color wherever you see this. we will use what comes default.